21/day01/src/main.rs

```diff
@@ -0,0 +1,41 @@
+fn main() {
+    let i = include_str!("input.txt");
```

Comment:
separate main.rs and lib.rs

Reply:
Rubbish! Separat in the main.rs what interacts with the outside (inclusive the result) und let all in the lib.rs that helps exclusively to solve the puzzle.